js/main.js

```diff
@@ -0,0 +1,174 @@
+  var $navH1 = document.querySelector('.navh1');
+  data.wrongPokemon.push(data.currentPokemon)
+  data.currentNumber++;
+  $navH1.textContent = "Question " + data.currentNumber
+  var $quizContainer = document.querySelector('.quizContainer');
+  $quizContainer.remove();
+  createQuizContainer();
+  generateFourRandomPokemonNumbers();
+  getPokemonPicture()
+  console.log(data);
+}
+
+function questionsAndTime() {
+  var $barRow = document.createElement('div');
+  $barRow.className = "bar";
+  var $quizContainer = document.querySelector('.quizContainer');
+  $quizContainer.prepend($barRow);
+  var $inRow = document.createElement('div');
+  $inRow.className = "in";
+  $barRow.appendChild($inRow);
+  var shuffledFour = _.shuffle(data.currentFour);
+  for (var i = 0; i < 4; i++) {
+    var $button = document.createElement('button');
+    $button.className = "white-button justify-center";
+    $button.textContent = allPokemonList[shuffledFour[i] - 1]
+    $quizContainer.appendChild($button);
+    $button.addEventListener('click', questionClick)
+  }
+}
+
+function questionClick() {
+  clearTimeout(tenSecondsBar);
+  var $quizContainer = document.querySelector('.quizContainer');
+  console.log(event.target.textContent);
+  if (data.currentNumber === 10) {
+    alert('goodjob')
+    //Checks
+    if (event.target.textContent === data.currentPokemon) {
+      var $dots = document.querySelectorAll('.col-tenth')
+      $dots[data.currentNumber - 1].textContent = ""
+      var $icon = document.createElement('img');
+      $icon.className = "icon"
+      $icon.setAttribute('src', 'images/pokeball.png')
+      $dots[data.currentNumber - 1].appendChild($icon)
+      data.correctPokemon.push(data.currentPokemon);
+    } else {
+      data.wrongPokemon.push(data.currentPokemon)
+    }
+  } else if (event.target.textContent === data.currentPokemon) {
+    console.log('NEXT QUESTION, remake DOM');
+    var $navH1 = document.querySelector('.navh1');
+    var $dots = document.querySelectorAll('.col-tenth')
+    $dots[data.currentNumber - 1].textContent = ""
+    var $icon = document.createElement('img');
+    $icon.className = "icon"
+    $icon.setAttribute('src', 'images/pokeball.png')
+    $dots[data.currentNumber - 1].appendChild($icon)
+    //Push current pokemon
+    data.correctPokemon.push(data.currentPokemon);
+    data.currentNumber++;
+    $navH1.textContent = "Question " + data.currentNumber
+    $quizContainer.remove();
+    createQuizContainer()
+    generateFourRandomPokemonNumbers();
+    getPokemonPicture();
+    console.log(data);
+  } else {
+    //WRONG ANSWERS
+    console.log('NEXT QUESTION, remake DOM');
+    var $navH1 = document.querySelector('.navh1');
+    data.wrongPokemon.push(data.currentPokemon)
+    data.currentNumber++;
+    $navH1.textContent = "Question " + data.currentNumber
+    $quizContainer.remove();
+    createQuizContainer();
+    generateFourRandomPokemonNumbers();
+    getPokemonPicture()
+    console.log(data);
```

Comment:
These exact lines are in the above conditional block too. Since you do it in both blocks, you should just do it once below the conditional.